drivers/gpio/gpio_mcux_igpio.c

```diff
@@ -18,57 +18,51 @@ struct mcux_igpio_config {
 };
 
 struct mcux_igpio_data {
+	/* gpio_driver_data needs to be first */
+	struct gpio_driver_data general;
 	/* port ISR callback routine address */
 	sys_slist_t callbacks;
 	/* pin callback routine enable flags, by pin number */
 	u32_t pin_callback_enables;
 };
 
+static int mcux_igpio_pin_interrupt_configure(struct device *dev,
```

Comment:
Since this function is not called by `mcux_igpio_configure` anymore we could skip the declaration.